pkg/resources/external_oauth_integration.go

```diff
@@ -93,10 +93,14 @@ var externalOauthIntegrationSchema = map[string]*schema.Schema{
 		ConflictsWith:    []string{"external_oauth_allowed_roles_list"},
 	},
 	"external_oauth_allowed_roles_list": {
-		Type:          schema.TypeSet,
-		Elem:          &schema.Schema{Type: schema.TypeString},
-		Optional:      true,
-		Description:   relatedResourceDescription("Specifies the list of roles that the client can set as the primary role.", resources.AccountRole),
+		Type:             schema.TypeSet,
+		Elem:             &schema.Schema{Type: schema.TypeString},
+		Optional:         true,
+		Description:      relatedResourceDescription("Specifies the list of roles that the client can set as the primary role.", resources.AccountRole),
+		DiffSuppressFunc: SuppressIfAny(
+		// TODO(SNOW-1517937): uncomment
```

Comment:
when? I see the linked issue is in progress?

Reply:
Ah, this ticket should be slightly different. For now, we have these lists optional, but all of them across external, custom and partner oauths are missing `NormalizeAndCompareIdentifiersInSet`. Let's do this as a follow-up.